src/app/pages/dashboard/widgets/system/common/widget-sys-info.scss

```diff
@@ -7,6 +7,44 @@
   height: 100%;
 }
 
+:host ::ng-deep {
```

Comment:
Seems like it works just fine without `::ng-deep`.

Reply:
now it's required, I updated `.product-image-placeholder {` which requires it.